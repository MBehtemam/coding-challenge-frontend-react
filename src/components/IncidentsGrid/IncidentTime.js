import styled from 'styled-components';

const IncidentTime = styled.time`
  color: yellow;
`;

export default IncidentTime;
