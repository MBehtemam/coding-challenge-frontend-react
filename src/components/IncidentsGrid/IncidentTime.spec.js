import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import IncidentTime from './IncidentTime';

describe('Test suits for Incident Time', () => {
  it('Should match to snapshot', () => {
    const tree = renderer
      .create(<IncidentTime datetime="2008-02-14 20:00">2008-02-14 20:00</IncidentTime>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
