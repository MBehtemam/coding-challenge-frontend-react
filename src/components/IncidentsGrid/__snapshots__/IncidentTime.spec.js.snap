// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Incident Time Should match to snapshot 1`] = `
.c0 {
  color: yellow;
}

<time
  className="c0"
>
   datetime="2008-02-14 20:00"
</time>
`;
